general/community-standards.md

```diff
@@ -0,0 +1,52 @@
+---
+uid: community-standards
+title: Jellyfin Community Standards
+---
+
+# Jellyfin Community Standards
+
+### Why?
```

Comment:
I think that deserves the same title style as the other items. And even if not, going from 1 to 3 to 2 might look a bit odd..
```suggestion
## Why?
```

Reply:
I picked that formatting specifically, since the "Why" is a little subsection of the main body, then the level 2 header is for the actual sections of the article. I could go to 4 there for less visual difference, but I find this style appealing personally.

Reply:
I just removed the header since the linter complains about this too. I don't personally see the issue, but it works fine as an intro paragraph too.